cmd/contour/contour.go

```diff
@@ -112,6 +122,7 @@ func newClient(kubeconfig string, inCluster bool) (*kubernetes.Clientset, *clien
 
 	client, err := kubernetes.NewForConfig(config)
 	check(err)
+
```

Comment:
nit: maybe use \n on line 127 and onwards for consistency?

Reply:
Sorry I don't follow your comment. I didn't change any of this short of moving from `kubernetes.ClientSet` to `kubernetes.Interface`. The `check(err)` stops execution of the program.